packages/rocketchat-lib/server/functions/addUserToRoom.js

```diff
@@ -1,4 +1,8 @@
 RocketChat.addUserToRoom = function(rid, user, inviter, silenced) {
+	//when user is registered, it does not have prpoerty username rather it just have name
```

Comment:
I am not sure why this is needed? This seems like a change which should not be required as it changes the core Rocket.Chat behavior.